src/viewer/core/ViewerWindow.h

```diff
@@ -802,26 +802,26 @@ class VIEWERCORE_API ViewerWindow : public ViewerBase
     static void PickFunctionSetSuccessFlag(void *, bool, const PickAttributes *);
     void HandlePick(void *ppi);
 
-    VisWindow           *visWindow;
-    ViewerPlotList      *plotList;
-    ViewerActionManager *actionMgr;
+    VisWindow           *visWindow {nullptr};
+    ViewerPlotList      *plotList {nullptr};
+    ViewerActionManager *actionMgr {nullptr};
```

Comment:
Why initializers in header when they are initialized in constructor?

Reply:
Fell swoop of initializing everything in the header regardless of what happens in the C file. Selective inits in the header are confusing so do them all.